nms/app/packages/magmalte/app/views/traffic/PolicyFlows.js

```diff
@@ -271,11 +294,11 @@ function PolicyFlowFields2(props: FieldProps) {
                           />
                         </AltFormFieldSubheading>
                       </Grid>
-                      <Grid item xs={12} sm={6}>
-                        <AltFormFieldSubheading label={'Destination Port'}>
+                      <Grid item xs={12}>
+                        <AltFormFieldSubheading label={''}>
                           <OutlinedInput
                             data-testid="udpDestination"
-                            placeholder="0"
+                            placeholder="Destination Port"
```

Comment:
Having the label non-empty lets the purpose of the field be seen at a glance, especially when coming back to edit the field. It's less of a problem when initially creating the policy, for sure. I disagree with moving the label text into the placeholder.